assets/locales/en.json

```
@@ -155,5 +149,27 @@
   "runGarbageCollectorErrored": {
     "title": "Garbage collector",
     "message": "The garbage collector run could not be completed successfully."
+  },
+  "downloadCidContentDialog": {
+    "title": "Download",
```

(Just an idea, feel free to ignore): 
what if we set expectations about what "Next" does via the dialog title?

```suggestion
    "title": "Download to a local directory",
```

---

+1 😊